cmd/scm_integrations.go

```diff
@@ -1,167 +1,168 @@
+	$ scalingo integrations
 
-	# See also commands 'scm-integrations-create', 'scm-integrations-delete', 'scm-integrations-import-keys'`,
+	# See also commands 'integrations-add', 'integrations-delete', 'integrations-import-keys'`,
 
 		Action: func(c *cli.Context) {
-			err := scm_integrations.List()
+			err := scmintegrations.List()
 			if err != nil {
 				errorQuit(err)
 			}
 		},
 		BashComplete: func(c *cli.Context) {
-			_ = autocomplete.CmdFlagsAutoComplete(c, "scm-integrations")
+			_ = autocomplete.CmdFlagsAutoComplete(c, "integrations")
 		},
 	}
 
-	scmIntegrationsCreateCommand = cli.Command{
-		Name:     "scm-integrations-create",
-		Category: "SCM Integrations",
+	integrationsAddCommand = cli.Command{
+		Name:     "integrations-add",
+		Category: "Integrations",
 		Flags: []cli.Flag{appFlag,
-			cli.StringFlag{Name: "url", Usage: "URL of the SCM integration"},
-			cli.StringFlag{Name: "token", Usage: "Token of the SCM integration"},
+			cli.StringFlag{Name: "url", Usage: "URL of the integration"},
+			cli.StringFlag{Name: "token", Usage: "Token of the integration"},
 		},
-		Usage: "Link your Scalingo account with your account on a SCM tool (such as github.com)",
-		Description: `Link your Scalingo account with your account on a SCM tool. After creating the link, you will be able to automatically deploy when pushing to your SCM repository, or create Review Apps for all pull requests created.
+		Usage: "Link your Scalingo account with your account on a tool such as github.com",
+		Description: `Link your Scalingo account with your account on a tool. After creating the link, you will be able to automatically deploy when pushing to your repository, or create Review Apps for all pull requests created.
 
 	For github.com:
-	$ scalingo scm-integrations-create github
+	$ scalingo integrations-add github
 
 	For gitlab.com:
-	$ scalingo scm-integrations-create gitlab
+	$ scalingo integrations-add gitlab
 
 	For GitHub Enterprise:
-	$ scalingo scm-integrations-create github-enterprise --url https://ghe.example.com --token personal-access-token
+	$ scalingo integrations-add github-enterprise --url https://ghe.example.com --token personal-access-token
 
 	For GitLab Self-hosted:
-	$ scalingo scm-integrations-create gitlab-self-hosted --url https://gitlab.example.com --token personal-access-token
+	$ scalingo integrations-add gitlab-self-hosted --url https://gitlab.example.com --token personal-access-token
 
-	# See also commands 'scm-integrations', 'scm-integrations-delete', 'scm-integrations-import-keys'`,
+	# See also commands 'integrations', 'integrations-delete', 'integrations-import-keys'`,
 
 		Action: func(c *cli.Context) {
 			if c.NArg() != 1 {
-				_ = cli.ShowCommandHelp(c, "scm-integrations-create")
+				_ = cli.ShowCommandHelp(c, "integrations-add")
 				return
 			}
 
-			link := c.String("url")
+			integrationURL := c.String("url")
 			token := c.String("token")
 			scmType := scalingo.SCMType(c.Args()[0])
```

Comment:
Do we want to try to get the type from the URL?

Reply:
Here you are in the `scm_integrations` file. So a user type one of the following commands:

```
scalingo integrations-add github
scalingo integrations-add gitlab
scalingo integrations-add github-enterprise --url https://ghe.example.com --token personal-access-token
scalingo integrations-add gitlab-self-hosted --url https://gitlab.example.com --token personal-access-token
```

How would you deduce the type from the URL?

Reply:
I thought of `scalingo integrations-add --url https://github.com` but yeah it's not really good.